userguide/intro.rst

```diff
@@ -232,6 +232,12 @@ These screen options have changed:
   fields have been added to
   :menuselection:`Tasks --> Cloud Sync Tasks --> Add Cloud Sync`.
 
+* The :guilabel:`NIC` and :guilabel:`Interface Name` fields in
+  :menuselection:`Network --> Interfaces --> Add Interface`
+  are preconfigured with the web interface NIC settings when configuring
+  the first interface. The UI also warns the user to configure the web
+  interface to prevent connection issues.
```

Comment:
Try to avoid saying "UI".  But also the web interface warning is only shown if the user tries to edit another interface first.  So:
"...interface. A warning is shown if changes are made to an additional interface before configuring the web interface."